commands/command_edit.cpp

```diff
@@ -269,6 +269,27 @@ DiveField EditWaterTemp::fieldId() const
 	return DiveField::WATER_TEMP;
 }
 
+// ***** Atmospheric pressure *****
```

Comment:
Copy & paste? Should probably read "Water type".

Reply:
Done